bindings/generator/api/os.py

```diff
@@ -0,0 +1,4 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+def get_os() -> str:
```

Comment:
what about a variant instead of a string ?

Reply:
We can use a variant instead, would you prefer that ?

> By the way, I find it strange to ask that question when you request change at the same time.

Reply:
Yes I'd rather have a variant 👍

Reply:
done